man/figures/moma-formula.svg

```diff
@@ -0,0 +1,91 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
What exactly is this? In general, I wouldn't think we'd want to commit images unless they are logos or similar.

Reply:
Hmmm.... Ok - this might not render right on CRAN or other backends though. I think a better long-term fix is to write the math in `R` using the plotmath functionality and autogenerate the graphic so things can be put together dynamically. I'll open an issue.